src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.FigureSupplier;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
```

Comment:
It's better not to leave such comments in the code. They carry no valuable information

Reply:
Fixed